packages/wallet/package.json

```diff
@@ -19,12 +19,12 @@
 		"contracts:deploy:test": "npx etherlime deploy --compile=false --file=./deployment/deploy-test.js",
 		"contracts:deploy": "npx etherlime deploy --compile=false",
 		"deployContracts": "NODE_ENV=development npx deploy-contracts",
-		"lint:check": "yarn run tslint --config tslint.json --project .",
-		"lint:write": "yarn run tslint --config tslint.json --project . --fix",
+		"lint:check": "yarn run tslint --config tslint.json  '{src,config,deployments,gateways,migrations,scripts}/**/*.{js,ts,tsx}'",
```

Comment:
Should these things be whitelisted like this? Is there any reason not to lint every `jsx?|tsx?` file?

Reply:
We want to avoid linting `node_modules`. However I think I can just exclude that rather than whitelisting everything.